repository-test/src/main/java/tech/ydb/yoj/repository/test/RepositoryTestSupport.java

```diff
@@ -30,13 +28,7 @@ public void setUp() {
 
     @After
     public void tearDown() {
-        clearDb(this.repository);
-    }
-
-    @SuppressWarnings({"unchecked", "rawtypes"})
-    public static void clearDb(Repository repo) {
-        var tables = repo.tables();
-        new StdTxManager(repo).tx(() -> tables
-                .forEach(table -> Tx.Current.get().getRepositoryTransaction().table((Class) table).deleteAll()));
+        repository.dropDb();
```

Comment:
is it possible to do it?

Reply:
You should probably keep the old logic, which deletes all data from the tables actually created (from repo.tables()), but **DOES NOT DROP THE TABLES** themselves. You should be able recreate the old logic just fine, because now repo.tables() returns a set of TableDescriptors